instance_test.go

```diff
@@ -0,0 +1,61 @@
+package thema_test
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/grafana/thema"
+	"github.com/grafana/thema/internal/txtartest/bindlin"
+	"github.com/grafana/thema/internal/txtartest/vanilla"
+	"github.com/grafana/thema/vmux"
+
+	"cuelang.org/go/cue/cuecontext"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInstance_Translate(t *testing.T) {
+	test := vanilla.TxTarTest{
+		Root: "./testdata/lineage",
+		Name: "core/instance/translate",
+	}
+
+	ctx := cuecontext.New()
+	rt := thema.NewRuntime(ctx)
+
+	test.Run(t, func(tc *vanilla.Test) {
+		if !tc.HasTag("multiversion") {
+			return
+		}
+
+		tName := strings.Replace(tc.Name(), t.Name()+"/", "", -1)
+
+		lin, lerr := bindlin.BindTxtarLineage(tc, rt)
+		require.NoError(tc, lerr)
+
+		for sch := lin.First(); sch != nil; sch = sch.Successor() {
```

Comment:
It takes a bit long (_locally it takes around ~50s_), but I think the exhaustiveness may worth it. Wdyt?

Reply:
Yup, we shouldn't avoid testing what ought to be tested just because it's slow. We'll get to those perf issues soon enough